### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_ScreenReader:Voice Over is not announcing the information within the colour graph.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5825]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS4.1.2,#MAS1.1.1; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

Environment Details:
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
Additional Details:
Tool Name: Voice Over
MAS Reference: MAS1.3.1,MAS4.1.2,MAS1.1.1
Repro Steps:
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab->Jobs->Click on any the jobs in the table.
4.Navigate inside the table by ctrl+opt+right/left arrow till graph.
5.Observe the announcement of the Voiceover.
Actual:
Voice-over is not announcing the information represented in the form of graph.
Expected:
Voice-over should announce the information represented in the form of graph.
**User Impact:**
User will be abide with the useful information represented in the form of graph.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference: MAS 1.3.1 MAS1.4.1 ,MAS1.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachment for Reference:**
[5825_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3247776/5825_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
